views/dashboard/exercisemetrics.html

```diff
@@ -22,27 +22,29 @@ <h3>{{=exercise_label}}</h3>
 			</div>
 			<div class="dash-section-content">
 				<div id="chart-exercise-attempts"></div>
-				<!--<div><b>Students who...</b>
```

Comment:
Everything from here to line 42 is commented out and has been for 3 years.  I propose we eliminate stuff that has been commented out.  The html is complicated enough with out cluttering it up with stuff that nobody even remembers why it's there.

Reply:
Yes! With pleasure!

(I hesitate to do this on my own, since this isn't my code...)